main/zoe/api/zoe-contract-facet.md

```diff
@@ -319,14 +319,15 @@ Returns the `brand` associated with the `issuer`.
 
 Returns the `issuer` of the `brand` argument.
 
-## zcf.getAmountMath(brand)
-- `brand` `{String}`
-- Returns: `{amountMath}`
+## zcf.getMathKind(brand)
+- `brand` `{Brand}`
+- Returns: `{MathKind}`
 
-Returns the `amountMath` object associated with the `brand` argument.
+Returns the `MathKind` associated with the `brand` argument.
 ```js
-const assetAmountMath = zcf.getAmountMath(assetAmount.brand);
+const quatloosMathKind = zcf.getMathKind(quatloosBrand);
 ```
+
 ## zcf.stopAcceptingOffers()
 - The contract requests Zoe to not accept offers for this contract instance. 
 It can't be called from outside the contract unless the contract explicitly makes it accessible.
```

Comment:
In line 367, we need to delete `maths:`. It will no longer exist

Reply:
Also need to fix lines: 68, 61, 120, and 154. There will be no `amountMath` associated with a ZCFMint